File: web/src/test/javascript/edu/dfci/cccb/mev/web/javascript/log.spec.js
define ([ 'log' ], function (log) {
  describe ("log", function () {
    it ("should log to error", function () {
      log.error ("error");
    });

    it ("should log to warn", function () {
      log.warn ("warn");
    });

    it ("should log to info", function () {
      log.info ("info");
    });

    it ("should log to debug", function () {
      log.debug ("debug");
    });
  });
});

File: common/domain/src/test/r/log.spec.R
suite ('Logger injectables', function () {
  it ('Should have injectable debug log',
      function ()
        inject (function (debug) expect (is.function (debug))));

  it ('Should have injectable info log',
      function ()
        inject (function (info) expect (is.function (info))));

  it ('Should have injectable warn log',
      function ()
        inject (function (warn) expect (is.function (warn))));

  it ('Should have injectable error log',
      function ()
        inject (function (error) expect (is.function (error))));
});
